src/rosdep2/catkin_packages.py

```diff
@@ -32,7 +34,21 @@ def find_catkin_packages_in(path, verbose=False):
         return _catkin_packages_cache[path]
     packages = find_packages(path)
     if type(packages) == dict and packages != {}:
-        package_names = [package.name for package in packages.values()]
+        package_names=[]
+        for package in packages.values():
+            package_names.append(package.name)
+            if package.replaces:
+                for replaced in package.replaces:
+                    if replaced.name in _catkin_replaced_packages:
+                        print("WARNING: '{0}' replaces '{1}', but '{1}' is already replaced by '{2}'".format(
+                            package.name, replaced.name,
+                            _catkin_replaced_packages[replaced.name]))
+                    else:
+                        _catkin_replaced_packages[replaced.name] = package.name
+            if package.conflicts:
+                for conflicted in package.conflicts:
+                    if conflicted.name not in _catkin_conflicted_packages:
+                        _catkin_conflicted_packages[conflicted.name] = package.name
```

Comment:
If they are both installed, then a pair of warning will be generated (A conflicts B, and B conflicts A). If only one of them is installed, the other one will be skipped.